app/src/androidTest/java/org/simple/clinic/MigrationAndroidTest.kt

```diff
@@ -289,4 +291,22 @@ class MigrationAndroidTest {
       assertThat(it.getInt(it.getColumnIndex("hasDiabetes"))).isEqualTo(falseAsInt)
     }
   }
+
+  @Test
+  fun migration_14_to_15() {
+    val db_v14 = helper.createDatabase(TEST_DB_NAME, 14)
+
+    // We need to do this here because Room does not create virtual tables for us.
+    db_v14.execSQL("""CREATE VIRTUAL TABLE "PatientFuzzySearch" USING spellfix1""")
```

Comment:
Do you intentionally use multi-line quotes for single lines? I can update this for you.

Reply:
I'm intentionally using multi-line here because I prefer using double-quotes for SQLite identifiers.